apps/web/app/layoutHOC.tsx

```diff
@@ -51,7 +51,7 @@ export function WithLayout<T extends Record<string, any>>({
         requiresLicense={requiresLicense || !!(Page && "requiresLicense" in Page && Page.requiresLicense)}
         nonce={nonce}
         themeBasis={null}
-        isThemeSupported={!!(Page && "isThemeSupported" in Page && Page.isThemeSupported)}
+        isThemeSupported={Page && "isThemeSupported" in Page ? (Page.isThemeSupported as boolean) : undefined}
```

Comment:
NIT: I think `isThemeSupported={Page?.isThemeSupported ?? undefined}` has the same effect here and is a lot more readable 
```suggestion
        isThemeSupported=isThemeSupported={Page?.isThemeSupported ?? undefined}
) : undefined}
```

Reply:
`Page?.isThemeSupported` will throw type error :( Feel free to make tweaks